docs/en/04_Changelogs/beta/4.7.0-beta1.md

```diff
@@ -10,6 +10,19 @@
 
 ## New features
 
+### Updated file usage table
+
+CMS users can view where a file is used by accessing the "Used On" tab in the file editor. This feature allows them to identify what DataObjects depend on the File.
+
+The following changes have been made to the used on table:
+- Data is now sourced from has_one and many_many relations instead of the ownership API
+- This means more data relationships should be shown than before
+- Uses the new RelatedDataService API in the framework module which can also be used for other non-file use cases
+- Support for linking ancestor relationships in the same table row, for instance showing the Page a Content Block is on
+- Ability to click on a row and navigate to the most relevant `CMSEditLink()`
+- Removal of draft badges
+- Styling tweaks such as reduced padding
+
```

Comment:
```suggestion
- Styling tweaks such as reduced padding

Your project specific DataObjects will automatically be displayed on this Used on tab. This may not always be desirable, especially when working with background DataObjects the user can not interact with directly. It is recommended you view the CMS Developer Guide for more information on this change, see the [File Usage section](docs/en/02_Developer_Guides/14_Files/07_File_Usage).
```

Reply:
Note I haven't tested that link. Dependant on this PR....

Reply:
Updated